samples/mesh/main.c

```diff
@@ -265,17 +261,7 @@ void main(void)
             frames = 0;
             last = now;
         }
-
-        /* Exit the demo after timeout */
-        if ((reboot_after > 0) && ((now-start) > reboot_after)) {
-            break;
-        }
     }
-
-    MmFreeContiguousMemory(alloc_vertices);
```

Comment:
May want to keep these even if they are unreachable so people know what cleanup would look like.

Reply:
Agree. keep this and `pb_kill`.
Then add a comment to tell people that this is not reachable before the cleanup code.